app/javascript/dashboard/routes/dashboard/settings/canned/Index.vue

```diff
@@ -2,7 +2,7 @@
   <div class="column content-box">
     <woot-button
       color-scheme="success"
-      class-names="button--fixed-right-top"
+      class-names="button--fixed-top"
```

Comment:
I see a lot of changes with `button--fixed-right-top` changed to `button--fixed-top`

Just verify if that it's working correctly for multiple screens, you can use https://responsively.app/ or https://polypane.app/ to test these out quickly

Reply:
This class is used for this button in the settings screens.
<img width="435" alt="image" src="https://user-images.githubusercontent.com/64252451/220059628-6391ca18-9106-43f8-87ea-953c4c15a617.png">

Here is the loom video.
https://www.loom.com/share/8715173e0d7f4d3a87367f7c4c2292c4